### ISSUE TITLE
Build Failure: Logstash 6.0 failing because of missing logstash-core (6.0.0.rc1) dependency

### ISSUE THREAD

**Issue by robbavey** [elastic/logstash#8236]
Logstash 6.0 builds are failing on Jenkins (https://logstash-ci.elastic.co/job/elastic+logstash+6.0+multijob-unix-compatibility/os=ubuntu/57/console):

```
15:11:43 Invoking bundler install...
15:12:29 Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8
15:12:34 Error Bundler::InstallError, retrying 1/10
15:12:34 An error occurred while installing logstash-core (6.0.0.rc1), and Bundler cannot continue.
15:12:34 Make sure that `gem install logstash-core -v '6.0.0.rc1'` succeeds before bundling.
```

**Comment by original-brownbear** [comment]
Funny enough, I get 

```sh
Bundler::GemNotFound: You have requested:
  rspec ~> 3.1.0

The bundle currently has rspec locked at 3.6.0.
```

here

**Comment by original-brownbear** [comment]
hmm the problem here seems to be that during `rake plugin:install-all` `logstash-filter-punct` is pulled, which requires plugins api `~ 1.0` (https://github.com/logstash-plugins/logstash-filter-punct/blob/master/logstash-filter-punct.gemspec#L23).
(at least that's what happens for me locally)

**Comment by robbavey** [comment]
Another failure 

https://logstash-ci.elastic.co/job/elastic+logstash+6.0+multijob-unix-compatibility/os=ubuntu/60/console

**Comment by original-brownbear** [comment]
@andrewvc you last touched the lock file so I'm asking you :)
Is it ok to fix this by adding plugins that pull in a plugins api version too old (`< 2.x`) to the skip list so that the install all task works out?

**Comment by original-brownbear** [comment]
@suyograo @jordansissel I guess you guys probably also know the answer to the question above? 
The problem here is simply that `rake install:plugin-all` is pulling plugins that are not compatible with the current plugins api version.
Ignore or fix plugins? :)

**Comment by jordansissel** [comment]
Do we still use install:plugin-all? Maybe we can just remove this task.

**Comment by original-brownbear** [comment]
@jordansissel hmm idk :) also please ignore my question here, I just realized I missread the ci bash script ... this is just an issue with installing the bundle for the QA acceptance tests.
Shouldn't be too hard to fix :)

**Comment by suyograo** [comment]
Fixed in #8267
